contracts/Recipes/RecipeV3.sol

```diff
@@ -108,6 +109,7 @@ contract RecipeV3 is IRecipe, Ownable {
     unitOracle oracle = unitOracle(0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6);
     uniV3Router uniRouter = uniV3Router(0xE592427A0AEce92De3Edee1F18E0157C05861564);
     IUniRouter sushiRouter = IUniRouter(0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506);
+    ICurveSwaps curveRouter = ICurveSwaps(0xD1602F68CC7C4c7B59D686243EA35a9C73B0c6a2);
```

Comment:
This address looks like Curve's mainnet `Swaps` contract, it should be the Polygon address.